bugbug/db.py

```diff
@@ -115,13 +115,19 @@ def upload(path):
 
 
 def last_modified(path):
+
+    if is_different_schema(path):
+        raise Exception("Last-Modified is not available")
+
     url = DATABASES[path]["url"]
-    last_modified = utils.get_last_modified(url)
+    last__modified = utils.get_last_modified(url)
```

Comment:
Why are you changing the names of the variables?

Reply:
Let's keep it the same name as before. We could change it, but it should be done in a separate PR (each PR should be self-contained)

Reply:
Ok

Reply:
Done